automata/fa/dfa.py

```diff
@@ -80,10 +82,30 @@ def __init__(
             initial_state=initial_state,
             final_states=final_states,
             allow_partial=allow_partial,
+            _digraph=self._get_digraph(transitions),
         )
+
+        self.reset_cache()
+
+    def reset_cache(self) -> None:
```

Comment:
@eliotwrobson I would kindly ask that the method be called `clear_cache()` rather than `reset_cache()`. I think the former has better semantics and is the more common verb to use in a caching context.

This requested change would include updating the verbiage in the docstrings, Markdown docs, and test function names for consistency, as well.

Reply:
Done here: https://github.com/caleb531/automata/pull/152/commits/61e16a660cdd8bb455adf081b0bc4e9d44fb03ac